extended/src/main/java/apoc/load/Jdbc.java

```diff
@@ -134,11 +147,23 @@ private Stream<RowResult> executeUpdate(String urlOrKey, String query, Map<Strin
                 throw sqle;
             }
         } catch (Exception e) {
-            log.error(String.format("Cannot execute SQL statement `%s`.%nError:%n%s", query, e.getMessage()),e);
-            String errorMessage = "Cannot execute SQL statement `%s`.%nError:%n%s";
-            if(e.getMessage().contains("No suitable driver")) errorMessage="Cannot execute SQL statement `%s`.%nError:%n%s%n%s";
-            throw new RuntimeException(String.format(errorMessage, query, e.getMessage(), "Please download and copy the JDBC driver into $NEO4J_HOME/plugins,more details at https://neo4j-contrib.github.io/neo4j-apoc-procedures/#_load_jdbc_resources"), e);
+            logsErrorAndThrowsException(e, query, log);
         }
+        return Stream.empty();
+    }
+
+    private static void logsErrorAndThrowsException(Exception e, String query, Log log) {
```

Comment:
:hackerino: 
ah già, se mettiamo void, poi devi fare `return Stream.empty();`
allora fai come stava prima, che questo ritorna `RuntimeException`
e sotto fai return new...

Ci cambi solo il nome del metodo in pratica